app-games/src/main/java/com/aptoide/android/aptoidegames/error_views/BottomSheetErrorViews.kt

```diff
@@ -0,0 +1,132 @@
+package com.aptoide.android.aptoidegames.error_views
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.defaultMinSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.verticalScroll
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.dp
+import cm.aptoide.pt.extensions.PreviewDark
+import com.aptoide.android.aptoidegames.AptoideGamesBottomSheet
+import com.aptoide.android.aptoidegames.R
+import com.aptoide.android.aptoidegames.drawables.icons.getGenericError
+import com.aptoide.android.aptoidegames.drawables.icons.getNoNetworkError
+import com.aptoide.android.aptoidegames.theme.AGTypography
+import com.aptoide.android.aptoidegames.theme.Palette
+
+@Composable
+fun BottomSheetErrorView(
```

Comment:
It might be better to have an error view that can receive a title and subtitle and then have a generic error view that uses it with defined copies. It would be similar to the normal error views